bench/Cargo.toml

```diff
@@ -8,12 +8,12 @@ publish = false
 
 [dependencies]
 quinn = { path = "../quinn" }
-tokio = { version = "0.2.0-alpha.6", default-features = false, features = ["rt-current-thread"] }
+tokio = { version = "0.2.1", features = ["macros"] }
```

Comment:
This is missing the features necessary for `tokio::spawn` to work, at minimum.

Reply:
Fixed.